projects/organization-management/src/app/components/cost-center-budget/cost-center-budget.component.html

```diff
@@ -12,39 +12,31 @@
         <span class="progress-display">{{ spentBudgetPercentage | percent }}</span>
         <ng-template #BudgetPopover>
           <div class="row">
-            <div class="col-md-6">
-              <label>{{ 'account.budget.label' | translate : { '0': costCenter.budgetPeriod } }}:</label>
+            <div class="col-md-6 mb-2">
```

Comment:
As discussed, I would use the utility class mb-2 because there is no real semantic class for the combined spacing "col-md-6 mb-2" where we need to style it in future for all labels and it is a very specific styling in the popover. The class "col-form-label" cannot be used here since it has too many other styling.